creator-node/src/services/stateMachineManager/stateReconciliation/SecondarySyncHealthTrackerConstants.ts

```diff
@@ -0,0 +1,19 @@
+// See prometheus.constants.js for reference on potential errors
+export const SYNC_ERRORS_TO_MAX_NUMBER_OF_RETRIES = Object.freeze({
+  failure_fetching_user_replica_set: 10,
+  failure_force_resync_check: 10,
+  failure_fetching_user_gateway: 10,
+  failure_delete_db_data: 3,
+  failure_delete_disk_data: 3,
+  failure_sync_secondary_from_primary: 10,
+  failure_db_transaction: 3,
+  failure_export_wallet: 10,
+  failure_import_not_consistent: 3,
+  failure_import_not_contiguous: 3,
+  failure_inconsistent_clock: 10,
+  failure_undefined_sync_status: 3,
+  default: 5
```

Comment:
flagging that we'll probably want to fine tune these and add other errors based on what we see in prod after release

Reply:
technically these errors only get added from the `recordFailure` call. these fields are from the prometheus constants so if more errors come up, there needs to be a manual update here

Reply:
agree these all seem reasonable for now and we can tweak later